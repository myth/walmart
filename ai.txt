INTRODUKSJON TIL KUNSTIG INTELLIGENS
Hvilke av følgende scenario er Single Agent...
.Sjakk
..Kryssord
.Ludo
..Sudoku
Hvilke av følgende scenario er Multi Agent...
..Sjakk
.Sudoku
.Kryssord
..Ludo
En agent som har sensorer vil inneha en fullt observerbar verden.
.True
..False
En delvis observerbar verden kan skyldes støy på sensorer, eller mangel på nok sensorer til å måle alle tilstander.
..True
.False
Natural Language Processing is a capability required in the Turing test.
..True
.False
Knowledge representation is a capability not required in the Turing test.
.True
..False
Automated reasoning is a capability required in the Turing test.
..True
.False
Machine learning is a capability required in the Turing test.
..True
.False
A rational agent is an agent which acts to achieve the best expected outcome based on what it knows, its functions and the environment.
..True
.False
PEAS is an acrynom for...
..Performance
.Probability
.Evaluation
..Environment
.Accountability
..Actuators
.Self-awareness
..Sensors
What do we call the environment if an agent does not have any sensors?
.Fully observable
..Unobservable
.Partially observable
What describes an agents actions in a deterministic environment?
.There is uncertainty about the outcome of an action.
..Any action has a single guaranteed effect without uncertainty.
.Any action has a single guaranteed effect, with the exception of possible effects from other agents.
What describes an agents action in a strategic environment?
.There is uncertainty about the outcome of an action.
.Any action has a single guaranteed effect without uncertainty.
..Any action has a single guaranteed effect, with the exception of possible effects from other agents.
What descibes an agents action in a stochastic encironment?
..There is uncertainty about the outcome of an action.
.Any action has a single guaranteed effect without uncertainty.
.Any action has a single guaranteed effect, with the exception of possible effects from other agents.
Episodic experience means that that an agents decision could affect all future decisions.
.True
..False
Sequential experience means that an agents decision could affect all future decisions.
..True
.False
A static environment stays unchanged until the agent makes a change.
..True
.False
A dynamic environment itself does not change with time, but the agents performance score does.
.True
..False
A semi dynamic environment itself does not change with time, but the agents performance score does.
..True
.False
A discrete-state environment has a finite number of distinct states.
..True
.False
Chess has a discrete set of percepts and actions.
..True
.False
A continuous state environment has has a discrete, finite amount of states.
.True
..False
An agent in a fully observable, unknown environment can infer the outcome of all actions.
.True
..False
An agent in a fully observable, known environment can infer the outcome of all actions.
..True
.False
A simple reflex agent selects it's actions based in the current precept. Ignoring previous precepts.
..True
.False
A model-based reflex agent selects it's actions based on one or more specified goals.
.True
..False
A utility based agent tries to maximize it's expected "happiness", e.g a chain of actions reaching the goal yielding the highest performance measure.
..True
.False
