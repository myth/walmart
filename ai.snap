ITGK
Hvilken tjeneste/applikasjon på Internett krever vanligvis IKKE lav tidsforsinkelse?
..On-demand Video.
.Interaktiv audio og video
.Online gaming.
.IP telefoni.
Hvilket alternativ er IKKE en del av fossefallsmetoden
.Kravanalyse og definisjon
.System- og programvaretesting
..Brukergrensesnittvurdering
.Integrasjon og systemtesting
For et problem av størrelsen n finnes fire algoritmer med forskjellig tids-kompleksitet. Hvilken vil bruke lengst tid (i verste fall) på store problemer.
.O(1) (konstant tid)
.O(n) (lineær tid)
.O(n2) (kvadratisk tid)
..O(2^n) (eksponentiell tid)
I følge Nyquist-regelen er samplefrekvensen for lyd
.halvparten av de frekvensene et menneske kan høre
.den samme som de frekvensene et menneske kan høre
..det dobbelte av de frekvensene et menneske kan høre
.3 ganger de frekvensene et menneske kan høre
Den riktige rekkefølgen fra minst til størst er
.giga, kilo, mega, tera
..kilo, mega, giga, tera
.tera, kilo, mega giga
.kilo, mega, tera, giga
ALU brukes i
.Instruction Fetch
..Instruction Execution
.Result Return
.Instruction Decode
Hvilken av disse forkortelsene er en kjent prosessmodell innen systemutvikling
.SCRUP
.UTML
..RUP
.RAM
En digital-til-analog konverterer (DAC)
..Endrer digital informasjon til analog
.Konverterer kontinuerlig lyd til digital lyd
.Konverterer lyd til et elektrisk signal
.Gir tilnærmede verdier
Hvilket av de følgende er ikke et høy-nivå språk
.Java
.C
..Assembly
.Visual Basic
Når er et sekvensielt søk effektivt?
.Når datamengden er sortert
.Når dataene er tall
.Sekvensielle søk er aldri effektive
..Når det det letes etter ligger tidlig i datamengden
Sangen «A little bit» er 3 minutter og 47 sekunder lang. Hvor mange bit trengs for å lagre den i stereo på en vanlig musikk-CD?
.1 411 200
.40 042 800
.84 672 000
..320 342 400
Hva er VPN
.En metode som brukes for å oversette logiske navn til et IP-nummer
.En del av TCP/IP spesifikasjonen
..En måte å etablere en trygg/kryptert kommunikasjonskanal mellom to maskiner
.En metode en internettleverandør bruker for å distribuere sensitivt innhold
En router er
..en datamaskin som forbinder flere nettverk
.et program som sender informasjonspakker mellom 2 datamaskiner
.et program som setter sammen informasjonspakkene til meldinger før den leveres til mottaker
.en datamaskin som er koblet på internett
Gitt navnelisten «Alice, Byron, Carol, Duane, Elaine, Floyd, Gene, Henry, Iris». Hvilken søkealgoritme vil finne Carol først (gjøre færrest sammenligninger)?
.Binærsøk
.Sekvensielt søk
.Begge vil finne Carol like raskt
..Svaret er avhengig av hvordan binærsøkalgoritmen er implementert
En byte hukommelse kan lagre
.en av 1024 forskjellige tall
.et ord
..et ASCII tegn
.en blokk
Hvor mange steg er det i Fetch/Execute syklusen
.3
.4
..5
.6
Programtelleren endres direkte av instruksjoner som kalles
.Add and Multiply
..Branch and Jump
.Input and Output
.Now and Next
Hvordan representeres det heksadesimale tallet A8 binært?
..10101000
.10010100
.11001000
.10001100
Forkortelsen DDOS i pensum står for…
.Digital Disk Operating System
.Double Density Optical Storage
..Distributed Denial Of Service
.Data Directory On Site
Hovedforskjellen mellom Boehm sin spiralmodell for programvareutvikling og andre, tidligere prosessmodeller var
..eksplisitt fokus på analyse og håndtering av risiko
.eksplisitt fokus på arbeidsmiljøet i programvarebedriftene
.eksplisitt fokus på gradvis kompetanseutvikling i prosjektteamet
.eksplisitt fokus på gradvis kompetanseutvikling i bedriftsledelsen
Person A skal sende en konfidensiell melding til person B. Hva slags krypteringsnøkler skal i så fall brukes?
.A krypterer med A sin private nøkkel, B dekrypterer med A sin offentlige nøkkel
.A krypterer med B sin private nøkkel, B dekrypterer med A sin offentlige nøkkel
..A krypterer med B sin offentlige nøkkel, B dekrypterer med B sin private nøkkel
.A krypterer med B sin offentlige nøkkel, B dekrypterer med A sin offentlige nøkkel
Brannmur (firewall) er en type sikkerhetsteknologi. Hva er den mest korrekte og relevante påstanden når det gjelder brannmurer og trusler fra såkalte trojanske hester ("Trojan Horses")?
.Brannmurer kan verne mot trojanske hester ved å hindre uventet internett-trafikk fra utsida og inn til et system.
..Brannmurer kan verne mot trojanske hester ved å hindre uventet trafikk fra innsiden og ut til internett.
.Brannmurer kan verne mot trojanske hester ved å advare brukerne mot å åpne falske epostmeldinger.
.Brannmurer gir IKKE vern mot trojanske hester. Bare antivirusprogramvare er effektivt mot trojanske hester.
Hva er en viktig forskjell mellom systemtesting og akseptansetesting?
.Systemtesting fokuserer på å finne feil i et program, mens akseptansetesting fokuserer på de delene som fungerer.
..Systemtesting bruker gjerne oppkonstruerte testdata mens akseptansetesting bruker data fra kunden som skal ha systemet.
.Systemtesting tester bare systemet modul for modul, mens akseptansetesting tester helesystemet i et.
.Systemtesting kan gjøres av personell som ikke kan programmere, mens akseptansetesting må gjøres av personell som også kan programmere, for å rette uakseptable feil
